mitodl/mitxpro
# Partner logo in certificate template

#### Pre-Flight checklist


#### What are the relevant tickets?
Issue #2406 

#### How should this be manually tested?
- Checkout to this branch
- Apply Migrations
- Check in CMS if certificate page have field `Partner Logo` to upload partner logo
- After logo is uploaded check in preview if the logo is present in the template
- Check there is not styling issue in certificate template

#### Screenshots (if appropriate)
<img width="1104" alt="Screenshot 2022-08-15 at 5 26 25 PM" src="https://user-images.githubusercontent.com/77275478/184639825-b65a359c-2eb2-4388-bf24-76eb8fa32a15.png">
<img width="1102" alt="Screenshot 2022-08-15 at 5 23 26 PM" src="https://user-images.githubusercontent.com/77275478/184639874-ed5078d2-491a-4606-9b72-c191de85a86a.png">
<img width="270" alt="Screenshot 2022-08-15 at 5 25 35 PM" src="https://user-images.githubusercontent.com/77275478/184639917-0af96686-edac-4c9b-87ec-b6e4867d6a73.png">
